portfolio/src/main/webapp/script.js

```diff
@@ -10,4 +10,23 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function printMessageFromServer(){
```

Comment:
update this similarly to [your other pull request](https://github.com/samuelsogbesan/step/pull/17/files)

Reply:
Added space between () and { with commit 418b853